tests/album-create.test.js

```diff
@@ -0,0 +1,43 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('create album', () => {
+    let artistId;
+    beforeEach(async () => {
+        const { status, body } = await request(app)
+          .post('/artists')
+          .send({ name: 'Kurt Cobain', genre: 'rock' });
+
+          expect(status).to.equal(201);
+          expect(body.name).to.equal('Kurt Cobain');
+          expect(body.genre).to.equal('rock');
+          artistId = body.id;
+    });
+
+    describe('/albums', () => {
+        describe('POST', () => {
+            it('creates a new album in the database', async () => {
+                const { status, body } = await request(app)
+                  .post(`/artists/${artistId}/albums`)
+                  .send(
+                  {
+                    name: 'Nevermind',
+                    year: 1991,
+                    artistId: artistId,
```

Comment:
You don't need to send the artistId in the body of the request as it is taken from the req.params in the route on line 23
```
```